uxarray/grid.py

```diff
@@ -372,6 +373,82 @@ def integrate(self, var_ds, quadrature_rule="triangular", order=4):
 
         return integral
 
+    # A one-time-used helper function that will only be used in the build_face_edges_connectivity map() function usage
+    def __replace_fill_val(self, row, index, rep_val):
+        row[index] = rep_val
+        return row
```

Comment:
Function not necessary with the comments below. 

Also, I suggest not using double `_` because it has a special purpose in Python: https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name

Reply:
Understood, but this function is meant to be a super private class function that will only be used once in the `Grid.build_face_edge_connectivity`

Reply:
I would of still stuck with a single `_` , but I understand where you are coming from

Reply:
Can you explain in more detail about why a single `_` will be more appropriate in this situation? Thanks

Reply:
I see what you mean, ignore what I said!